src/vttLib/__main__.py

```diff
@@ -53,10 +53,12 @@ def main(args=None):
     parser_dumpfile.add_argument("outfile", nargs="?", metavar="OUTPUT.ttx")
     parser_dumpfile.set_defaults(func=vttLib.vtt_dump_file)
 
+    parser_mergefile.add_argument("--keep-cvar", action="store_true", default=False)
```

Comment:
```suggestion
    parser_mergefile.add_argument("--keep-cvar", action="store_true")
```

`store_true` has the side-effect of `default=False`.